modules/tinymce/src/plugins/table/main/ts/ui/Helpers.ts

```diff
@@ -5,73 +5,64 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { HTMLElement } from '@ephox/dom-globals';
-import { Arr, Fun, Obj, Strings } from '@ephox/katamari';
+import { Types } from '@ephox/bridge';
+import { Element as DomElement, HTMLElement, Node } from '@ephox/dom-globals';
+import { Arr, Obj, Strings, Fun } from '@ephox/katamari';
 import { Css, Element } from '@ephox/sugar';
+import DOMUtils from 'tinymce/core/api/dom/DOMUtils';
 import Editor from 'tinymce/core/api/Editor';
-import Tools from 'tinymce/core/api/util/Tools';
-
 import * as Styles from '../actions/Styles';
-import { shouldStyleWithCss, getDefaultStyles, getDefaultAttributes } from '../api/Settings';
-import { Types } from '@ephox/bridge';
+import { getDefaultAttributes, getDefaultStyles, shouldStyleWithCss } from '../api/Settings';
 
 /**
  * @class tinymce.table.ui.Helpers
  * @private
  */
 
-const buildListItems = (inputList, itemCallback, startItems?): Types.SelectBox.ExternalSelectBoxItem[] => {
-  const appendItems = (values, output?: Types.SelectBox.ExternalSelectBoxItem[]) => {
-    output = output || [];
-
-    Tools.each(values, (item) => {
-      const menuItem: any = { text: item.text || item.title };
-
-      // TODO TINY-2236 - this implies table_class_list supports nested menus, but bridge doesn't, so this either needs to be supported or deleted
-      if (item.menu) {
```

Comment:
we haven't implemented submenus in T5 so deleted, left a comment about the ticket that says we should do it

Reply:
I suggest logging / linking the tickets, so when we do 2236 we know we need to address both tables and links (I assume it'll be easy to do both once there's a bridge API)

Reply:
Good idea. I've added just a list of plugins that need updating, but feel free to expand on it further.

Reply:
Link and table are the only ones I know of